app/Channel.php

```diff
@@ -38,4 +38,19 @@ public function episode()
     {
         return $this->hasMany('Suyabay\Episode');
     }
+
+    public function scopePendingEpisodes($query, $id)
+    {
+        return $query->where('channels.id', '=', $id)
+                    ->join('episodes', 'channels.id', '=', 'episodes.channel_id')
```

Comment:
**Code Quality-Nitpick**
@andela-tolotin Quoting integers in PHP is a ticking time bomb, that might cause future bugs to creep out, please fix this and all other similar occurrences.